tutorials/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -285,6 +272,48 @@ As of Godot 4.0, exporting .NET projects is supported for desktop platforms
 (Linux, Windows and macOS). Other platforms will gain support in future 4.x
 releases.
 
+Common errors
+-------------
+
+You might encounter the following error when trying to modify some values in Godot
```

Comment:
Consider linking to Microsoft's documentation for CS1612 https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs1612.

Reply:
It is linked below, I can move it up here though.

Reply:
Oh, sorry, I totally missed up :pray:. It's fine then.